docs/CLI.md

```diff
@@ -0,0 +1,142 @@
+# covalent cli
+
+## Overview
+
+CLI (Command Line Interface) for Covalent.
+
+
+### Usage
+
+To run the covalent tools cli do the following:
+
+* Clone this repository
+* npm i
+* npm run build
+* npm install -g
+
+#### The following commands are available in the CLI
+
+#### Help Usage
+```
+cov -h
+  Usage: cov [options]
+
+
+  Options:
+
+    -V, --version           output the version number
+    -v, --all-versions      output the version information
+    -g, --gen-e2e-report    generate the e2e coverage report
+    -s, --start-e2e-proxy   start the e2e proxy
+    -e, --e2e-for-coverage  runs the end to end tests against the e2e proxy
+    serve                   starts the ng server
+    serve-e2e               starts the ng server with flags needed to run e2e coverage
+    e2e                     runs the end to end tests against local code base
+    lint                    runs the linter
+    test                    runs the unit tests
+    build                   builds the code
+    -h, --help              output usage information
+```
+
+#### Versions
+* Prints out all versions numbers for Covalent CLI, Covalent Version, Angular, Angular Material Version, and Angular CLI for the directory you are currently in
+```
+cov -v
+```
+
+#### ng Commands
+* Same as `ng serve --proxy-config proxy.conf.json`
+```
+cov serve
```

Comment:
these are so specific to our particular use case.  Is it possible to allow custom modifiers or flags, other than default state?  Perhaps a future enhancement?

Reply:
I think that would be a great future enhancement after we get the baseline 👍